### PR TITLE
Run code formatter on errors_test.exs

### PR DESCRIPTION
**Pull request by thebrianemory**

### CODE REVIEW

File: lib/elixir/test/elixir/kernel/errors_test.exs [hunk 1]
```diff
@@ -549,16 +582,20 @@ defmodule Kernel.ErrorsTest do
...
       "nofile:1: module Certainly.Doesnt.Exist is not loaded and could not be found",
       'import Certainly.Doesnt.Exist'
+    )
   end
 
   test "module imported from the context it was defined in" do
-    assert_eval_raise CompileError,
+    assert_eval_raise(
+      CompileError,
       ~r"nofile:4: module Kernel.ErrorsTest.ScheduledModule.Hygiene is not loaded but was defined.",
```
Comment (josevalim) [inline lib/elixir/test/elixir/kernel/errors_test.exs]: Please extract the message to a variable on its own line.

File: lib/elixir/test/elixir/kernel/errors_test.exs [hunk 2]
```diff
@@ -567,10 +604,12 @@ defmodule Kernel.ErrorsTest do
...
       end
       '''
+    )
   end
 
   test "module imported from the same module" do
-    assert_eval_raise CompileError,
+    assert_eval_raise(
+      CompileError,
       ~r"nofile:3: you are trying to use the module Kernel.ErrorsTest.ScheduledModule.Hygiene which is currently being defined",
```
Comment (josevalim) [inline lib/elixir/test/elixir/kernel/errors_test.exs]: Please extract the message to a variable on its own line.

File: lib/elixir/test/elixir/kernel/errors_test.exs [hunk 3]
```diff
@@ -579,26 +618,32 @@ defmodule Kernel.ErrorsTest do
...
       end
       '''
+    )
   end
 
   test "already compiled module" do
-    assert_eval_raise ArgumentError,
+    assert_eval_raise(
+      ArgumentError,
       "could not call eval_quoted with argument Record " <>
```
Comment (josevalim) [inline lib/elixir/test/elixir/kernel/errors_test.exs]: Please extract the message to a variable on its own line.

File: lib/elixir/test/elixir/kernel/errors_test.exs [hunk 4]
```diff
@@ -676,160 +733,169 @@ defmodule Kernel.ErrorsTest do
...
-      ':ok ?a'
-    assert_eval_raise SyntaxError,
-      "nofile:1: syntax error before: \"?\\\\s\"",
-      ':ok ?\\s'
-    assert_eval_raise SyntaxError,
-      "nofile:1: syntax error before: \"?す\""
-      ':ok ?す'
+    assert_eval_raise(SyntaxError, "nofile:1: syntax error before: \"?a\"", ':ok ?a')
+    assert_eval_raise(SyntaxError, "nofile:1: syntax error before: \"?\\\\s\"", ':ok ?\\s')
+    assert_eval_raise(SyntaxError, "nofile:1: syntax error before: \"?す\"")
```
Comment (josevalim) [inline lib/elixir/test/elixir/kernel/errors_test.exs]: This seems to be a bug caught by the formatter, the next line should have been the last argument for this function.

File: lib/elixir/test/elixir/kernel/errors_test.exs [hunk 5]
```diff
@@ -676,160 +733,169 @@ defmodule Kernel.ErrorsTest do
...
+    ':ok ?す'
   end
 
   test "invalid \"fn do expr end\"" do
-    assert_eval_raise SyntaxError,
+    assert_eval_raise(
+      SyntaxError,
       "nofile:1: unexpected token \"do\". Anonymous functions are written as:\n\n" <>
```
Comment (josevalim) [inline lib/elixir/test/elixir/kernel/errors_test.exs]: Please extract the message to a variable on its own line.

File: lib/elixir/test/elixir/kernel/errors_test.exs [hunk 6]
```diff
@@ -676,160 +733,169 @@ defmodule Kernel.ErrorsTest do
...
+      def foo(n) when is_number(n)
+    end
+    ''')
   end
 
   test "invalid args for bodyless clause" do
-    assert_eval_raise CompileError,
+    assert_eval_raise(
+      CompileError,
       ~r"nofile:2: only variables and \\\\ are allowed as arguments in definition header.",
```
Comment (josevalim) [inline lib/elixir/test/elixir/kernel/errors_test.exs]: Please extract the message to a variable on its own line.

File: lib/elixir/test/elixir/kernel/errors_test.exs [hunk 7]
```diff
@@ -676,160 +733,169 @@ defmodule Kernel.ErrorsTest do
...
-      rescue_stacktrace("""
-      defmodule Kernel.ErrorsTest.MacrosErrorStacktrace do
-        defmacro sample(num), do: num + :foo
-        def other, do: sample(1)
-      end
-      """)
+    assert [
+             {:erlang, :+, [1, :foo], _},
+             {Kernel.ErrorsTest.MacrosErrorStacktrace, :sample, 1, _} | _
+           ] =
```
Comment (josevalim) [inline lib/elixir/test/elixir/kernel/errors_test.exs]: Please break this pattern in two. Something like this:

```
assert [plus_2, sample_1 | _] =
  rescue_stacktrace("""....""")
assert {:erlang, :+, [1, :foo], _} = plus_2
assert {Kernel.ErrorsTest.MacrosErrorStacktrace, :sample, 1, _} = sample_1
```

File: lib/elixir/test/elixir/kernel/errors_test.exs [hunk 8]
```diff
@@ -676,160 +733,169 @@ defmodule Kernel.ErrorsTest do
...
-      """)
+    assert [
+             {
+               Kernel.ErrorsTest,
+               :__before_compile__,
+               [%Macro.Env{module: Kernel.ErrorsTest.MacrosCompiledCallback}],
+               _
+             }
+             | _
+           ] =
```
Comment (josevalim) [inline lib/elixir/test/elixir/kernel/errors_test.exs]: Same here, please break the stacktrace match apart.

File: lib/elixir/test/elixir/kernel/errors_test.exs [hunk 9]
```diff
@@ -676,160 +733,169 @@ defmodule Kernel.ErrorsTest do
...
     rescue
       ArgumentError ->
-        stack = System.stacktrace
-        assert [{:erlang, :apply, [1, :foo, []], []},
-                {__MODULE__, :bad_remote_call, 1, [file: _, line: _]} | _] = stack
+        stack = System.stacktrace()
+
+        assert [
+                 {:erlang, :apply, [1, :foo, []], []},
+                 {__MODULE__, :bad_remote_call, 1, [file: _, line: _]} | _
```
Comment (josevalim) [inline lib/elixir/test/elixir/kernel/errors_test.exs]: Same here.

File: lib/elixir/test/elixir/kernel/errors_test.exs (hunk 3 above)
Comment (josevalim) [inline lib/elixir/test/elixir/kernel/errors_test.exs]: You can probably remove the concatenation.

File: lib/elixir/test/elixir/kernel/errors_test.exs (hunk 9 above)
Comment (thebrianemory) [reply lib/elixir/test/elixir/kernel/errors_test.exs]: Something like this? Also I'm bad at variable names so if you have better suggestions I'm all ears. :)
```
        assert [apply, bad_remote_call | _] = stack
        assert {:erlang, :apply, [1, :foo, []], []} = apply
        assert {__MODULE__, :bad_remote_call, 1, [file: _, line: _]} = bad_remote_call
```

File: lib/elixir/test/elixir/kernel/errors_test.exs (hunk 8 above)
Comment (thebrianemory) [reply lib/elixir/test/elixir/kernel/errors_test.exs]: Am I on the right track with this one?
```
    assert [callback | _] =
             rescue_stacktrace("""
             defmodule Kernel.ErrorsTest.MacrosCompiledCallback do
               Module.put_attribute(__MODULE__, :before_compile, Kernel.ErrorsTest)
             end
             """)

    assert {
             Kernel.ErrorsTest,
             :__before_compile__,
             [%Macro.Env{module: Kernel.ErrorsTest.MacrosCompiledCallback}],
             _
           } = callback
```

File: lib/elixir/test/elixir/kernel/errors_test.exs (hunk 5 above)
Comment (thebrianemory) [reply lib/elixir/test/elixir/kernel/errors_test.exs]: Like so?
```
    message =
      "nofile:1: unexpected token \"do\". Anonymous functions are written as:\n\n" <>
        "    fn pattern -> expression end\n\n" <> "Syntax error before: do"

    assert_eval_raise(SyntaxError, message, 'fn do :ok end')
```

File: lib/elixir/test/elixir/kernel/errors_test.exs (hunk 4 above)
Comment (thebrianemory) [reply lib/elixir/test/elixir/kernel/errors_test.exs]: Setting the last line to
`assert_eval_raise(SyntaxError, "nofile:1: syntax error before: \"?す\"", ':ok ?す')`
causes the test to fail.

I can change it to
`assert_eval_raise(SyntaxError, "nofile:1: syntax error before: \"?す\"")`
and the test will pass.

How would you like me to have it for the purpose of this PR?

**Comment by josevalim** [comment]
I will do those fixes locally, thanks.

**Pull request merged**